src/webgpu/api/validation/texture/bgra8unorm_storage.spec.ts

```diff
@@ -108,3 +111,95 @@ if the feature bgra8unorm-storage is not enabled.
 
     t.testCreateShaderModuleWithBGRA8UnormStorage(shaderType, false);
   });
+
+g.test('configure_storage_usage_on_canvas_context_without_bgra8unorm_storage')
+  .desc(
+    `
+Test that it is invalid to configure a GPUCanvasContext with GPUStorageBinding usage and a GPUDevice
```

Comment:
How about `Test that it is invalid to configure a GPUCanvasContext to 'GPUStorageBinding' usage with 'bgra8unorm' format on a GPUDevice not enabled 'bgra8unorm-storage'. `